src/translator/response_builder.cpp

```diff
@@ -65,7 +65,7 @@ void ResponseBuilder::buildTranslatedText(Histories &histories,
 
     Result result = onebest[0]; // Expecting only one result;
     Words words = std::get<0>(result);
-    auto targetVocab = vocabs_->back();
+    auto targetVocab = vocabs_->target();
```

Comment:
Statements of these nature are now redundant, part of the intent of the refactor was to directly pass `vocabs_->target()` at call-site than temporarily assign to variable to describe better.